src/viking.js

```diff
@@ -1,11 +1,56 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor (health, strength){
+        this.health = health
+        this.strength = strength
+
+    }
+    attack(){
```

Comment:
Recuerda que un método funciona como una función dentro de la clase. No necesitas definir una función dentro de este para poder regresar un resultado. ej. 
`attack( ){ 
return this.strength
}`